tracing-example/pom.xml

```diff
@@ -48,6 +46,39 @@
         <debugJvmArgs/>
     </properties>
 
+    <profiles>
+        <profile>
+            <id>wildfly27</id>
+            <properties>
+                <version.org.wildfly>27.0.1.Final</version.org.wildfly>
+            </properties>
+            <dependencies>
+                <dependency>
+                    <groupId>org.jboss.resteasy</groupId>
+                    <artifactId>resteasy-tracing-api</artifactId>
+                    <version>${version.org.jboss.resteasy.extensions}</version>
+                </dependency>
+            </dependencies>
+        </profile>
+        <profile>
+            <id>wildfly28</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <properties>
+                <version.org.wildfly>28.0.0.Beta1</version.org.wildfly>
+            </properties>
+            <dependencies>
+                <dependency>
+                    <groupId>org.jboss.resteasy</groupId>
+                    <artifactId>resteasy-tracing-api</artifactId>
+                    <version>${version.org.jboss.resteasy.extensions}</version>
+                    <scope>provided</scope>
+                </dependency>
+            </dependencies>
+        </profile>
```

Comment:
I think this profile should be removed. We can keep the property we had and change it to 28.0.0.Beta1 like you had previously.